hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AConfiguration.java

```diff
@@ -511,6 +513,34 @@ public void testConfOptionPropagationToFS() throws Exception {
     assertOptionEquals(updated, "fs.s3a.propagation", "propagated");
   }
 
+  @Test(timeout = 10_000L)
+  public void testConnectTtlPropagation() throws Exception {
+    Configuration config = new Configuration();
```

Comment:
We should unset the property here in case someone has already set this to some value in their auth-keys.xml so that the default value test passes even if some other value is set.